sunpy/map/tests/test_map_factory.py

```diff
@@ -263,6 +270,14 @@ def test_map_list_urls_cache():
             'https://github.com/sunpy/data/raw/main/sunpy/v1/AIA20110607_063305_0094_lowres.fits']
     sunpy.map.Map(urls)
 
+@pytest.mark.remote_data
+def test_map_list_uri():
+    """
+    Test for reading from URI (AWS S3).
+    """
+    uri_list = ['s3://gov-nasa-hdrl-data1/sdo/aia/20100513/0094/sdo_aia_h2_20100513T000000_0094_v1.fits',
+            's3://gov-nasa-hdrl-data1/sdo/aia/20100513/0131/sdo_aia_h2_20100513T000400_0131_v1.fits']
```

Comment:
I think rather than using these data we should use the [sunpy sample data](http://data.sunpy.org/) these are mirrored in s3 and public: I was testing with this: `m = sunpy.map.Map("s3://data.sunpy.org/sunpy/AIA20110607_065843_0193_cutout.fits", fsspec_kwargs={"anon": True})`